src/components/InfoBox/InfoBox.stories.tsx

```diff
@@ -40,10 +40,8 @@ const InfoBoxExample = () => {
           priority for G20 countries, philanthropy and the private sector.
         </p>`}
       </Text>
-      <Listing>
-        <Contact name="John Smith" />
-        <Contact name="Bertie Basset" />
-      </Listing>
+      <Contact name="John Smith" />
```

Comment:
I don't think it is possible to add a contact component within list. So it is not possible for them to be wrapped by `<Listing>`

(I based it on [the page I created](https://develop.wellcome-corporate-fe.org.uk/what-we-do/our-work/listing-options) and on the teams page where there is a lot of contact us within InfoBox and non of it is wrapped within `<Listing>` component, e.g. [International cultural programmes team](https://wellcome.org/who-we-are/teams/international-cultural-programmes-team)

Reply:
It may have been the case once upon a time but looking at the CMS I don't think it is possible.